cmake/FreecivDependencies.cmake

```diff
@@ -116,7 +116,13 @@ add_subdirectory(dependencies/sol2)
 include(FreecivBackward)
 
 # Compression
-find_package(KF6Archive REQUIRED)
+include(FetchContent)
+FetchContent_Declare(KF6Archive
+  URL https://download.kde.org/stable/frameworks/6.13/karchive-6.13.0.tar.xz
+  URL_HASH SHA256=3c9b5dcf3abdfe2761e2153d70d9d667f1ff0fd2f6c80addba7549da954fcc90
+  FIND_PACKAGE_ARGS NAMES KF6Archive
+)
+FetchContent_MakeAvailable(KF6Archive)
```

Comment:
This seems out of place. Are you thinking of compiling kf6archive as an external dep like we do the font for the time being? If so, this should be a separate PR

Reply:
Sorry, it leaked from work I've been doing in parallel. Will remove.